24.04/ubuntu-debug/Dockerfile

```diff
@@ -13,7 +13,6 @@ FROM ghcr.io/cybozu/ubuntu:${TAG}
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
         apache2-utils \
-        awscli \
```

Comment:
`awscli` is used and it should be included in containers.
How about installing it following the official procedures?
Additionally, you might need to run `apt-get install unzip`.

```sh
&& curl "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m).zip" -o "awscliv2.zip" \
&& unzip awscliv2.zip \
&& ./aws/install \
&& rm -f awscliv2.zip
```

Reply:
I took it.